app/models/provider.rb

```diff
@@ -130,23 +130,23 @@ def current_accredited_courses
 
   validates :email, email_address: true, if: :email_changed?
 
-  validates :provider_name, length: { maximum: 100 }, on: :update
+  validates :provider_name, presence: true, length: { maximum: 100 }
 
-  validates :provider_code, uniqueness: { scope: :recruitment_cycle }
+  validates :provider_code, presence: true, uniqueness: { scope: :recruitment_cycle }
 
-  validates :provider_type, presence: true
+  validates :provider_type, presence: { message: "Provider type can't be blank" }
 
-  validates :telephone, phone: { message: "^Enter a valid telephone number" }, if: :telephone_changed?
+  validates :telephone, phone: { message: "Enter a valid telephone number" }, if: :telephone_changed?
```

Comment:
Can we add this to the translations too please?

Reply:
It overrides the default phone validator message which has the caret, so it should initially have been written `phone: true` , I think we are raising another ticket to deal with all the no longer needed caret mesages and gem